cf/i18n/resources/zh_Hans.all.json

```diff
@@ -2730,6 +2735,11 @@
       "modified": false
    },
    {
+      "id": "Show information for a stack (a stack is a pre-built file system, including an operating system, that can run apps)",
+      "translation": "Show information for a stack (a stack is a pre-built file system, including an operating system, that can run apps)",
```

Comment:
@SrinivasChilveri, since, I assume, you speak Chinese (correct?) might be cool to also submit the translation...  Thanks.